lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.js

```diff
@@ -31,6 +31,21 @@ testRule({
 			code: 'a { -webkit-transition-delay: 0.5s; transition-duration: 2s; transition-property: top; transition-timing-function: ease; }',
 			description: 'one of properties is prefixed',
 		},
+		{
+			code: 'div { inset: 0; }',
+		},
+		{
+			code: 'div { inset: 1px 2px; }',
+		},
+		{
+			code: 'div { inset: 1px 2px 3px; }',
+		},
+		{
+			code: 'div { inset: 1px 2px 3px 4px; }',
+		},
+		{
+			code: 'div { top: 0; right: 0; bottom: 0; }',
```

Comment:
[suggest] Could you use `a` selector instead of `div`?

```suggestion
			code: 'a { inset: 0; }',
		},
		{
			code: 'a { inset: 1px 2px; }',
		},
		{
			code: 'a { inset: 1px 2px 3px; }',
		},
		{
			code: 'a { inset: 1px 2px 3px 4px; }',
		},
		{
			code: 'a { top: 0; right: 0; bottom: 0; }',
```